src/Tasks.UnitTests/CreateItem_Tests.cs

```diff
@@ -261,6 +264,151 @@ public void AdditionalMetaDataOverwriteExisting()
             Assert.True(success);
             Assert.Equal("SomeOverwriteValue", t.Include[0].GetMetadata("MyMetaData"));
         }
+
+        /// <summary>
+        /// Logs error when encountering wildcard drive enumeration during task item creation.
+        /// </summary>
+        [Fact]
+        public void WildcardDriveEnumerationTaskItemLogsError()
+        {
+            using (var env = TestEnvironment.Create())
+            {
+                ChangeWaves.ResetStateForTests();
+                env.SetEnvironmentVariable("MSBUILDFAILONDRIVEENUMERATINGWILDCARD", "1");
+                BuildEnvironmentHelper.ResetInstance_ForUnitTestsOnly();
+
+                MockEngine engine = new MockEngine();
+                CreateItem t = new CreateItem()
+                {
+                    BuildEngine = engine,
+                    Include = new ITaskItem[] { new TaskItem(@"\**") },
+                };
+
+                t.Execute().ShouldBeFalse();
+                engine.Errors.ShouldBe(1);
+                engine.AssertLogContains("wildcard that results in drive enumeration");
+
+                ChangeWaves.ResetStateForTests();
+            }
+        }
+
+        /// <summary>
+        /// Logs warning when encountering wildcard drive enumeration during task item creation.
+        /// Note: Leave this test commented out, as it requires running "subst z: <drive>:\<path>"
+        /// on the command line for it to pass. Otherwise, the following test results in an
+        /// UnauthorizedAccessException while attempting to access the root of the drive.
```

Comment:
Instead of commenting it out, please file a bug describing the problem and mark this test with `[ActiveIssue(url_to_your_bug)]`. Likewise for all tests that can't succeed in the default state.